### PR TITLE
Lock wheels -> master

### PR DESCRIPTION
**Pull request by greenblitz4590**

### CODE REVIEW

File: src/main/java/edu/greenblitz/tobyDetermined/commands/swerve/LockWheels.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+	
+	PigeonGyro pigeon;
+
+
+	public LockWheels(){}
+
+	@Override
+	public void initialize(){
+		pigeon = swerve.getPigeonGyro();
+		new RotateAllWheelsToAngle(Math.PI / 2 - pigeon.getYaw()).schedule();
```
Comment (roygold8) [inline src/main/java/edu/greenblitz/tobyDetermined/commands/swerve/LockWheels.java:19]: Not crucial but better to use math deg to radians

File: src/main/java/edu/greenblitz/tobyDetermined/commands/swerve/RotateAllWheelsToAngle.java [hunk 2]
```diff
@@ -1,21 +1,33 @@
...
 import edu.wpi.first.math.kinematics.SwerveModuleState;
 
 public class RotateAllWheelsToAngle extends SwerveCommand {
-	private double targetAngle;
+	private final double targetAngle;
+	private final double threshold = Math.toRadians(4);
+	
+	private int timeOutCounter =0;
 	
 	public RotateAllWheelsToAngle(double targetAngleInRads) {
```
[the remark this answers is not in the document]
Comment (greenblitz4590) [reply src/main/java/edu/greenblitz/tobyDetermined/commands/swerve/RotateAllWheelsToAngle.java:13]: all the code is in rads, easier that way

File: src/main/java/edu/greenblitz/tobyDetermined/commands/swerve/LockWheels.java (hunk 1 above)
Comment (greenblitz4590) [reply src/main/java/edu/greenblitz/tobyDetermined/commands/swerve/LockWheels.java:19]: done!

File: src/main/java/edu/greenblitz/tobyDetermined/subsystems/swerve/KazaSwerveModule.java [hunk 3]
```diff
@@ -139,6 +139,18 @@ public SwerveModuleState getModuleState() {
 		return new SwerveModuleState(getCurrentVelocity(), new Rotation2d(this.getModuleAngle()));
 	}
 	
+	@Override
+	public boolean isAtAngle(double targetAngleInRads, double errorInRads) {
+		double currentAngleInRads = getModuleAngle() % Math.PI;
```
[the remark this answers is not in the document]
Comment (greenblitz4590) [reply src/main/java/edu/greenblitz/tobyDetermined/subsystems/swerve/KazaSwerveModule.java:144]: toRadians(180)?

[the remark this answers is not in the document]
Comment (greenblitz4590) [reply src/main/java/edu/greenblitz/tobyDetermined/subsystems/swerve/KazaSwerveModule.java:144]: looks less readble, leaving as is.

**Pull request merged**